contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -92,6 +93,48 @@ contract LidoOracle is ILidoOracle, AragonApp {
     bytes32 internal constant LAST_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.lastCompletedEpochId");
     bytes32 internal constant PREV_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.prevCompletedEpochId");
 
+    /*
+    If we use APR as a basic reference for increase, and expected range is between 1 and 10% APR.
+    So the granularity step for yearly APR is 1%
+    daily_granularity = 0.01 / 365 = 2.74e-05
+    Currently used basisPoints (1e-4) is unable to express such microscopic ratio.
+    So we should take another popular metric to express this - PPM or 1e-6
+    daily_granularity_in_ppm = 2.74e-05 / 1e-6 = 27 (pretty good)
+    with 100% buffer for future spikes it will be 50.
+    May be changed by Gov.
+    */
+    bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION =
+        keccak256("lido.LidoOracle.allowedBeaconBalanceAnnualRelativeIncrease");
+    uint256 public constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE = 50;  // PPM
```

Comment:
I think there is no need to make it public: this will create a new getter for it. Let's make them all `internal`?

Reply:
it's the other way round, better to default to public data unless we have a reason to hide

Reply:
reverted to public